### PR TITLE
add a readme as index to render screenshots page on github

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: screenshots/Readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Login
```
Comment (bastelfreak) [inline screenshots/Readme.md:1]: Usually we have it as README.md, should we rename the file?

**Comment by oneiros** [comment]
Wonderful idea. But please keep in mind that the screenshots are very much out-of-date. And they will be even more so once #216 has been merged.

**Comment by rwaffen** [comment]
@oneiros good point. let's open a ticket to remind ourself about updateing the images, once #216 is finished.

File: screenshots/Readme.md (hunk 1 above)
Comment (rwaffen) [reply screenshots/Readme.md]: thx for the hint

**Review by bastelfreak — APPROVED** [review]

**Comment by rwaffen** [comment]
issue is #222

**Pull request merged**